assets/js/buttons-examples.js

```diff
@@ -509,6 +509,15 @@ $(document).ready(function () {
           classes: "thumbsup-btn",
           text: "",
         },
+        {
```

Comment:
Place this in the correct alphabetical order under "Icon"

Reply:
Thanks for the feedback. I've placed it correctly now.